pop_tools/fill.py

```diff
@@ -112,7 +152,7 @@ def _iterative_fill_POP_core(nlat, nlon, var, fillmask, missing_value, tol, ltri
     done = False
     iter = 0
```

Comment:
`iter` is a Python built-in object. To avoid namespace collision, we should rename this to something along these lines:

```suggestion
    _iter = 0
```